test/files/pos/CharSequence-implicits.scala

```diff
@@ -0,0 +1,4 @@
+object Test extends App {
+  Array('a', 'b'): CharSequence
+  IndexedSeq('a', 'b'): CharSequence
+}
```

Comment:
All of our existing JUnit tests are in the `scala` package under `test/junit/scala`. But yeah, I _could_ be the pioneer here. We'd need to establish a convention. `test/junit/notscala`? `test/junit/scalauser`? And document the convention. And think about whether existing tests ought to be reclassified. I'm not sure whether I feel like tackling that right now. How about if I open a scala-dev ticket on it and add a comment to this one test...? Curious what the rest of the team thinks.